CMakeLists.txt

```diff
@@ -4,17 +4,40 @@ project (SyclSTL)
 enable_testing()
 
 option(PARALLEL_STL_BENCHMARKS "Build the internal benchmarks" OFF)
+option(USE_COMPUTECPP "Use ComputeCPP" ON)
 
-message(STATUS " Path to SYCL implementation: ${COMPUTECPP_PACKAGE_ROOT_DIR} ")
 message(STATUS " Path to CMAKE source directory: ${CMAKE_SOURCE_DIR} ")
+set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/Modules/)
 
+if (USE_COMPUTECPP)
+	message(STATUS " Using ComputeCpp CMake")
 
-set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/Modules/)
+	message(STATUS " Path to ComputeCpp implementation: ${COMPUTECPP_PACKAGE_ROOT_DIR} ")
+
+	include(FindOpenCL)
+	include(FindComputeCpp)
+
+	include_directories("${COMPUTECPP_INCLUDE_DIRECTORY}")
+
+else()
+	message(STATUS " Using triSYCL CMake")
 
-include(FindOpenCL)
-include(FindComputeCpp)
+	add_compile_options("-std=c++14") 
+
+	set(TRISYCL_INCLUDE_DIR "/usr/include" CACHE STRING "Path to triSYCL/include")
+	set(BOOST_COMPUTE_INCLUDE_DIR  "/usr/include" CACHE STRING "Path to Boost-compute/include")
+
+	message(STATUS "Path to triSYCL/include = " ${TRISYCL_INCLUDE_DIR})
```

Comment:
If we go this way, I guess it would be `"Path to triSYCL include directory="` sine /include has no meaning on some OS.